src/app/src/sensorUtils.js

```diff
@@ -122,14 +124,25 @@ export function transformSensorDataToRatings(sensorData) {
     const sensorRatings = VARIABLES.reduce((acc, variable) => {
         const { lower, upper } = sensor.HealthyRanges[variable];
         const variableValue = sensorData[variable];
+        // A variable is in fair condition if it is within 10% of the edges
+        // of the upper or lower ends of the healthy range. 0.1 is added below
+        // due to the inRange function not being inclusive.
+        const isVariableNearEdgeOfHealthyRange =
+            inRange(variableValue, lower, (upper - lower) / 10 + lower + 0.1) ||
```

Comment:
We talked about this in slack because math is hard, but the lodash `inRange` quirks of inclusivity and exclusivity I find confusing! It might serve us and our future selves better to write our own `inRangeInclusive` helper function.